### ISSUE TITLE
Design user settings page.

### ISSUE THREAD

**Issue by thequbit** [mediapublic/mediapublic#103]
We'll need some kind of page for users to manage settings.  We'll also need to identify what settings are on this page.

**Comment by nkkl** [comment]
Let's start with just managing their personal information - email, password, profile image, social media links, etc.

**Comment by thequbit** [comment]
So I think we've locked onto Twitter as our login method for the moment.  So no need for password.  We can just take their profile image from Twitter for the MVP.
- Email
- Facebook
- Twitter ( we have this already form their login )
- Instagram
- Periscope
- Personal Website

We also need a mechanism for linking a `User` to an `Organization`.  We haven't really talked about that yet.  I'll open a ticket.

I think that list is good to start.  Thoughts?

**Comment by nkkl** [comment]
This sounds like a great set of information to manage. Given the number of social media platforms we're supporting, we should allow users to pick, say, 2 primary points of contact so that we can have a verbose view and a truncated view. 6 ways to contact someone is overbearing, and they will likely have opinions about which plaftorms they'd prefer to be contacted on.

For example, I would emphasize my Twitter account (# 1) and Personal Website (# 2) in my profile, while someone else might put their Facebook and Periscope.

**Comment by thequbit** [comment]
I'll think about the best way to create those in the database.  I'm thinking a separate table for "ContactMethods" or something like that might make sense, and each row as a "preferred" flag.

**Comment by nkkl** [comment]
Makes sense. Then we can also be a little more flexible about what they can add, right?

**Comment by thequbit** [comment]
Exactly.  We'll "own" the list, but it will be nice and easy to add new "contact points".

**Comment by GabeIsman** [comment]
This page should probably include a short bio as well, or we pull the twitter bio.

**Comment by thequbit** [comment]
I think we can pull the twitter bio pretty easily for the MVP.

**Comment by melodykramer** [comment]
Agreed that Twitter bio will suffice.

**Comment by nkkl** [comment]
- Should have a way to look up and request to join an organization

**Comment by GabeIsman** [comment]
Really just a list of fields at this point, but here is the user settings page:
<img width="1426" alt="screenshot 2016-01-05 10 59 22" src="https://cloud.githubusercontent.com/assets/697801/12124671/9308fd76-b39b-11e5-9015-814d9cb44edc.png">
<img width="1422" alt="screenshot 2016-01-05 10 59 31" src="https://cloud.githubusercontent.com/assets/697801/12124674/9876ee94-b39b-11e5-8a36-508070cae2c1.png">

The profile view (non-editing) is unimplemented.
